study-builder/tenants/testboston/pages/login.html

```diff
@@ -20,7 +20,9 @@
 <![endif]-->
 
 <script src="https://cdn.auth0.com/js/lock/11.5.2/lock.min.js"></script>
+<script src="##BASE_URL##/assets/auth0-login-ar.js"></script>
 <script src="##BASE_URL##/assets/auth0-login-ht.js"></script>
+<script src="##BASE_URL##/assets/auth0-login-pt.js"></script>
```

Comment:
These are hosted on the frontend app. See PR https://github.com/broadinstitute/ddp-angular/pull/317.